contrib/go-pg/pg/pg_go_test.go

```diff
@@ -0,0 +1,53 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package pg
+
+import (
+	"context"
+	"testing"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/mocktracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	"github.com/go-pg/pg/v10"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSelect(t *testing.T) {
```

Comment:
Can you add a `TestImplements` something like:
```go
func TestImplementsHook(t *testing.T) {
    var _ pg.QueryHook = (*queryHook)(nil)
}

Reply:
I added this one.